workload/src/Samsung.Tizen.Build.Tasks/CheckApiVersion.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+using Samsung.Tizen;
+
+namespace Samsung.Tizen.Build.Tasks
+{
+    public class CheckApiVersion : Task
+    {
+        [Required]
+        public string ManifestApiVersion { get; set; }
+
+        [Required]
+        public string TargetFrameWorkIdentifier { get; set; }
+
+        [Required]
+        public string TargetPlatformIdentifier { get; set; }
+
+        [Required]
+        public string TargetFrameworkVersion { get; set; }
+
+        [Required]
+        public string TargetPlatformVersion { get; set; }
+
+        [Required]
+        public ITaskItem[] SupportedAPILevelList { get; set; }
+
+        string ApiVersion { get; set; }
+
+        public override bool Execute()
+        {
+            if (string.IsNullOrEmpty(ManifestApiVersion))
+            {
+                Log.LogError("ManifestApiVersion is Invalid {0}", ManifestApiVersion);
+                return false;
+            }
+
+            if (TargetFrameWorkIdentifier == "tizen")
```

Comment:
As far as I know, TFI will be `Tizen`, not `tizen`. please check when TFM is `tizen40`.

Reply:
Fixed